### DISCUSSION TITLE
[Feature]: Other placeable ignots

### DISCUSSION THREAD

**Discussion by GamingHQ** [Digitalroot-Valheim/Digitalroot.Valheim.GoldBars#4] [Ideas]
### Problem

Good morning.

My apologies for not being able to place anything in discussions. This however is due to the fact the only owners of a repository can open one where others can reply towards.

I have an idea, but I don't know if this would be possible. Would it be an option to add the ability to place unused ingots as well? iron, copper, black metal, and such, or would there be already such mod? I'm new to valheim, so I don't know much about trustworthy websites besides nexus mods. 

happy weekend wishes and please, stay safe!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Digitalroot** [comment]
> My apologies for not being able to place anything in discussions. This however is due to the fact the only owners of a repository can open one where others can reply towards.

Thank you for letting me know. I'll have to fix that.

**Reply by Digitalroot** [reply]
Give it a try and LMK if you are able to create new discussions.

**Reply by GamingHQ** [reply]
Yes, I see an option now called "new discussion " as well.

**Comment by Digitalroot** [comment]
Created #5 to track it.
